tracer/src/Datadog.Trace/IAST/StackWalker.cs

```diff
@@ -13,7 +13,24 @@ namespace Datadog.Trace.Iast;
 internal static class StackWalker
 {
     public static readonly string[] ExcludeSpanGenerationTypes = { "Datadog.Trace.Debugger.Helpers.StringExtensions" };
-    public static readonly string[] AssemblyNamesToSkip = { "Datadog.Trace", "System.Security.Cryptography", "System.Security.Cryptography.Primitives", "System.Security.Cryptography.Algorithms", "System.Security.Cryptography.Csp" };
+    public static readonly string[] AssemblyNamesToSkip =
+    {
+        "Datadog.Trace",
```

Comment:
Have you considered the use of wildcards here?

Reply:
I have but, for now, I think that they are not needed for performance resaons. In the future, wildcards might be useful and it might help us to filter many assemblies with just one wildcard expression.